.github/workflows/backend-ci-cd.yml

```
@@ -0,0 +1,97 @@
+      - name: Run Integration Tests for ${{ matrix.service }}
+        working-directory: apps/backend/${{ matrix.service }}
+        run: |
+          mvn verify -DskipUnitTests=true
+      - name: Stop and remove Docker containers
+        run: |
+          docker compose -f docker-compose.test.yaml down
+
+  coverage-report:
+    runs-on: ubuntu-latest
+    needs: [ unit-tests, integration-tests ]
+    steps:
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)